CRM/Chfunds/Upgrader.php

```diff
@@ -38,6 +38,19 @@ public function upgrade_1300() {
     return TRUE;
   }
 
+  public function upgrade_1400() {
+    $this->ctx->log->info('Applying update 1.4');
+    $sql = "
+      ALTER TABLE `civicrm_ch_contribution_batch` ADD COLUMN `campaign_id` INT NOT NULL
```

Comment:
@seamuslee001 can you also add alter query to set DEFAULT NULL property for ch_fund and fund column ? i.e.

```
"ALTER TABLE `civicrm_ch_contribution_batch` ADD COLUMN `campaign_id` INT NOT NULL,
 CHANGE `ch_fund` `ch_fund` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL DEFAULT NULL,
 CHANGE `fund` `fund` INT(11) NULL DEFAULT NULL;"
```